gather_data: merge job and test gathering (#58)
tarantool/multivac: docs/gather_data.rst

    $ # see data for the last 12 hours
    $ ./multivac/gather_job_data.py --since 12h

Writing data to JSON file

> -------------------------

Author: done

> To write data to JSON file, use ``--format json`` option. If you add `-t` option,
> script will write data about tests too:

Reviewer: You've picked the right structure in the first sentence. Use it in the second as well.

task → solution. task2 → solution2

```suggestion
To write data to a JSON file, use the ``--format json`` option.
To gather data about tests as well, add the ``--tests`` or ``-t`` option:
```

Author: done

> .. code-block:: console

Author: done

    {
    "8374189766": {

>     "job_id": 8374189766,
>     "workflow_run_id": 3061077644,
>     "job_name": "out_of_source",
>     "branch": "master",
>     "commit_sha": "416500fed508968d6d890eb3ec3620ef954d1d0a",
>     "conclusion": "success",
>     "queued_at": "2022-09-15T14:02:44Z",
>     "started_at": "2022-09-15T14:02:56Z",
>     "completed_at": "2022-09-15T14:08:45Z",
>     "platform": "amd64",
>     "runner_label": [
>       "ubuntu-20.04-self-hosted"
>     ],

Reviewer: this part is nested, so it should be indented. Just use any JSON formatter.

```suggestion
      "job_id": 8374189766,
      "workflow_run_id": 3061077644,
      "job_name": "out_of_source",
      "branch": "master",
      "commit_sha": "416500fed508968d6d890eb3ec3620ef954d1d0a",
      "conclusion": "success",
      "queued_at": "2022-09-15T14:02:44Z",
      "started_at": "2022-09-15T14:02:56Z",
      "completed_at": "2022-09-15T14:08:45Z",
      "platform": "amd64",
      "runner_label": [
        "ubuntu-20.04-self-hosted"
      ],
```

Author: done

    "runner_name": "ghacts-shared-8-16-n17",
    "runner_version": "2.296.2"
    "tests": [
    {
      "name": "box/tx_man.test.lua",
      "configuration": null,
      "status": "fail",
      "test_number": 0
    },
    {
      "name": "replication/qsync_advanced.test.lua",
      "configuration": "memtx",
      "status": "fail",
      "test_number": 1

>     }]}}

Author: Certain test data dict, list of tests, job id's dict, dict of jobs. All the braces are matches

Writing data to InfluxDB
------------------------

To write data to InfluxDB, you need to set credentials as environmental

> variables (see `.env-example` file). You can add them to the `.env` file and

Reviewer: ```suggestion
variables (see the ``.env-example`` file). You can add them to the ``.env`` file and
```

Author: User can just set environmental variables in any other way, in the script we don't use `.env` file, just call the environmental variables

activate with the command:

.. code-block:: console

    $ source .env && export $(cut -d= -f1 .env)

> Script uses different buckets for job data and test data. There are the data
> structure for job bucket:

Reviewer: Double 'data' can be reduced.

```suggestion
Script uses different buckets for job and test data.
The job bucket receives data in the following structure:
```

Author: done

.. code-block:: console

    {measurement: failure_type or 'success',
     tags: {
        job_id
        job_name
        workflow_run_id
        branch
        commit_sha
        platform
        runner_label
        runner_version
        runner_name
        conclusion },
     fields: {'value': 1},
     time: time the job started at, in nanoseconds}

> And there are data structure for test bucket:

Reviewer: ```suggestion
The test data bucket receives data in the following structure:
```

Author: done

.. code-block:: console

    {measurement: test_name,
     tags: {
        job_id
        configuration
        job_name
        commit_sha
        test_number
        },
     fields: {'value': 1},

>      time: time the job started at, in nanoseconds}

Author: fixed
